docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -239,18 +216,7 @@ EOF
 +
 endif::[]
 
-* Deploy OpenStackDataPlaneNodeSet:
-+
-Make sure that ovn-controller settings configured in the OpenStackDataPlaneNodeSet are the same as were set in the compute nodes before adoption.
-This configuration is stored in the "external_ids" column in the "Open_vSwitch" table in ovsdb and can be checked with command:
-+
-----
-ovs-vsctl list Open .
-...
-external_ids        : {hostname=standalone.localdomain, ovn-bridge=br-int, ovn-bridge-mappings="datacentre:br-ctlplane", ovn-chassis-mac-mappings="datacentre:1e:0a:bb:e6:7c:ad", ovn-encap-ip="172.19.0.100", ovn-encap-tos="0", ovn-encap-type=geneve, ovn-match-northd-version=False, ovn-monitor-all=True, ovn-ofctrl-wait-before-clear="8000", ovn-openflow-probe-interval="60", ovn-remote="tcp:ovsdbserver-sb.openstack.svc:6642", ovn-remote-probe-interval="60000", rundir="/var/run/openvswitch", system-id="2eec68e6-aa21-4c95-a868-31aeafc11736"}
-...
-----
-In above example bridge mappings are set as "datacentre:br-ctlplane" and it has to be set in the OpenStackDataPlaneNodeSet CR also.
+. Deploy the `OpenStackDataPlaneNodeSet` CR. Ensure that the bridge mappings are set as "datacentre:br-ctlplane".
```

Comment:
the exact value of the mappings depends on the adopted cluster setup. The previous version was talking about an "example", and we should be explicit in the new version too that this is just an example, and that the actual value should reflect **actual** configuration. (Below in the patch, line 397+, it may become more clear, but better be explicit here too.)